src/obb/models/game_info.cljc

```diff
@@ -1,23 +1,32 @@
 (ns ^{:added "0.1.0" :author "Pedro Pereira Santos"}
   obb.models.game-info
-  "This model represents a game between two players")
+  "This model represents a game between two players"
+  (:require [schema.core :as s]))
 
-(defn create
+(def IdSchema s/Str)
+
+(def GameInfo
```

Comment:
I named it `GameInfo` because we already have a `Game` concept on the rules side and we might need that extra vocabulary in the future.